.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
@danbev Oh yes, I saw the failing test and wasn't being able to open the travis url here, so I thought the setup was wrong, sorry!

looks like some weirdness around `const` without `'use strict';`